### ISSUE TITLE
Offset when beginning to draw (probably related to custom controls)

### ISSUE THREAD

**Issue by z0d14c** [Leaflet/Leaflet.draw#653]
Apologies for creating so many issues, but I'm working on them and I would like to document them / make them known as I do so.

Likely related to our use of custom controls in a lefthand tray, however still worth mentioning as it could affect anyone else using custom controls.

There seems to be a bug where an offset is created (a distance between the cursor and where the actual markers are made) when beginning to draw. This means you constantly have a big gap between your cursor and where you are drawing. I believe this is related to the fact that the UI which triggers the initial draw constructor is located off of the map, so a rogue initial click event initializes the marker far away from the marker.

**Comment by ddproxy** [comment]
Seeing as this is centered around custom code and placement of the tray, I'd appreciate a fiddle or jsbin that, even if it doesn't actually cause the problem, illustrates the conditions that may permit the issue.

**Comment by z0d14c** [comment]
fair enough, ill see if i can get to it. i think it likely has to do with the transparent mousemarker not following reliably

**Comment by jziggas** [comment]
Just ran into same issue. Have Leaflet.draw buttons as well as a custom button created with Leaflet.EasyButton. Unable to reliably reproduce though. Here is a [gif of it happening](http://recordit.co/dPeoDhGYMz).

**Comment by dankarran** [comment]
I have seen this in places too, and I think it might be happening when the map div doesn't have `position: relative;` set on it in the CSS. Adding that seems to fix the issue for me.
